### PR TITLE
read all .continuous files

### PR DESCRIPTION
**Pull request by rishidhingra**
fixes issue where openephysrawio reader ignores aux and adc .continuous files specified by the current/old open ephys format.

### CODE REVIEW

File: neo/rawio/openephysrawio.py [hunk 1]
```diff
@@ -87,13 +88,12 @@ def _parse_header(self):
...
             all_last_timestamps = []
             all_samplerate = []
-            for continuous_filename in info['continuous'][seg_index]:
+            for chan_id, continuous_filename in enumerate(info['continuous'][seg_index]):
                 fullname = os.path.join(self.dirname, continuous_filename)
                 chan_info = read_file_header(fullname)
 
                 s = continuous_filename.replace('.continuous', '').split('_')
                 processor_id, ch_name = s[0], s[1]
-                chan_id = int(ch_name.replace('CH', ''))
```
Comment (samuelgarcia) [inline neo/rawio/openephysrawio.py]: I don't anderstand this.

In neo rawio, the chan_id is something that match the original file format. Sometimes is is one based, sometimes can contains gaps.

Here, with enumerate it is more a chan_index.

In rawio the chan_index is the position, inside the "sig_channels" vector.

**Comment by samuelgarcia** [comment]
Hi @rishidhingra.
Thank you for the patch.
I think I don't anderstand it. Sorry.
Could you send me trhe file that trigegr the bug ?
And make a description of the problem with more detail please ?

**Comment by rishidhingra** [comment]
Hi Samuel,

Let me try explaining better. The bottom line is that there are more .continuous files produced by the openephys format than just the ones from the probes (which are named like 100_CH1.continuous, etc.), in particular the 100_ADC1.continuous files are produced when an experimenter uses the breakout board to record signals from other devices in parallel to the probe data. There are also files with an AUX prefix that can come from the accelerometers on some headstages (for recording animal position in awake recordings).

My patch accepts any of these prefixes, rather than only the CH (MEA) channel prefix. You can reproduce the bug by simply renaming a few .continuous files with the other prefixes (ADC or AUX). 

Apologies for not giving a more detailed explanation initially.

Best,
Rishi

________________________________

File: neo/rawio/openephysrawio.py (hunk 1 above)
Comment (samuelgarcia) [reply neo/rawio/openephysrawio.py]: Could we go back to chan_id even if chan_id can in some case have multiple entries.
 ```
            chan_str = re.split(r'(\d+)', s[1])[0]
            chan_id = int(ch_name.replace(chan_str, ''))
 ```
Here it is a channel index which is redundant to the channel position.

**Comment by samuelgarcia** [comment]
Hi @rishidhingra.
Ok I have read the PR.
It is Ok for me.

It would be would very good to have a dataset (very light)  [here](https://gin.g-node.org/NeuralEnsemble/ephy_testing_data).

Could you:
  * fix pep8 
  * go back to 
       ```
       chan_str = re.split(r'(\d+)', s[1])[0]
       chan_id = int(ch_name.replace(chan_str, ''))
       ```

     Instead  of chan_id becoming a global index.
     It is cool for the end user to retrieve the same number as they have on the device.
     Even if we can have 2 chan_id=0 (CH0 and AUX0)

So we could include, in the l release 0.8.

**Comment by samuelgarcia** [comment]
Hi @rishidhingra.
I have made some change directly on your branch could you have a look ?

There was a mix between chan_id and chan_index.
Now chan_id is clearly the XX CHXX so it is not unique when there is AUX.

And the chan_index is the global count index.

is it OK for you ?

**Comment by samuelgarcia** [comment]
@rishidhingra : for me this PR is ready to merge.
Could you make a last check on your dataset ?

**Pull request merged**